src/main/reactjs/src/tests/cypress/integration/clerk/meeting_dates.spec.ts

```diff
@@ -87,6 +83,21 @@ describe('MeetingDatesPage', () => {
     onMeetingDatesPage.expectAddButtonDisabled();
   });
 
+  it('should show an error toast when trying to add a meeting date, but a generic API error occurs', () => {
```

Comment:
selkeämpi ? should show a generic API error toast when trying to add a meeting date

Reply:
samalla voisi päivittää 

should not let user to add another meeting date with the same date as an existing one

-->

should not duplicate meeting dates

Reply:
> selkeämpi ? --> should show a generic API error toast when trying to add a meeting date

Voin vaihtaa noinkin. Mietin vaan alun perin että teksti kuvaisi juurikin sitä, että backend palauttaa geneerisen virheen eikä niinkään esim. virheen siitä, että tehtiin post-pyyntö duplikaatille päivämäärälle, josta tulee oma virheensä.